susemanager-utils/susemanager-sls/salt/packages/redhatproductinfo.sls

```diff
@@ -18,7 +18,7 @@ oraclerelease:
 amazonrelease:
   cmd.run:
     - name: cat /etc/system-release
-    - onlyif: test -f /etc/system-release
+    - onlyif: test -f /etc/system-release -a ! -L /etc/system-release
```

Comment:
I'm quite scared about this as I think this can be unreliable:
- this might be true now, but not in the future
- this might be true for other linux distributions, not only amazon linux

Wouldn't it make more sense to look into the `system-release` file and just do a simple grepping, for instance? We could use `file.find` modules with grep parameter (e.g. `^NAME.\*Amazon)`. Quite fishy, but I still think it's much more reliable.